notebook/static/notebook/less/cell.less

```diff
@@ -21,6 +21,11 @@ div.cell {
             border-color: transparent;
         }
     }
+    
+    &.marked {
+        border-left-color: blue;
```

Comment:
Another point, let's remove the top-left and bottom right corner rounding when the cell is marked. This will give contiguous block of marked cells a nice look.